test/PowerShellEditorServices.Test/Language/LanguageServiceTests.cs

```diff
@@ -217,9 +227,24 @@ public async Task LanguageServiceFindsReferencesOnCommandWithAlias()
                 await this.GetReferences(
                     FindsReferencesOnBuiltInCommandWithAlias.SourceDetails);
 
-            Assert.Equal(6, refsResult.FoundReferences.Count());
-            Assert.Equal("Get-ChildItem", refsResult.FoundReferences.Last().SymbolName);
-            Assert.Equal("ls", refsResult.FoundReferences.ToArray()[1].SymbolName);
+            SymbolReference[] foundRefs = refsResult.FoundReferences.ToArray();
+#if CoreCLR
+            // `ls` is not an alias on *nix...
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                Assert.Equal(6, foundRefs.Length);
+                Assert.Equal("ls", foundRefs[1].SymbolName);
```

Comment:
Is it possible to just not use `ls` in our tests so we don't have to have ifdefs?